plugins/outputs/http/README.md

```diff
@@ -0,0 +1,36 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`
```

Comment:
This section will be modified to detail how the http plugin works.

Thanks. sparrc